@shared/api/internal.ts

```diff
@@ -35,6 +35,7 @@ import { sendMessageToBackground } from "./helpers/extensionMessaging";
 import { getIconUrlFromIssuer } from "./helpers/getIconUrlFromIssuer";
 import { getDomainFromIssuer } from "./helpers/getDomainFromIssuer";
 import { stellarSdkServer } from "./helpers/stellarSdkServer";
+import BigNumber from "bignumber.js";
```

Comment:
this should actually throw a linting error. I'll fix the husky pre-commit hook for this